src/components/Views/MapView/Map/Map.js

```diff
@@ -37,15 +37,14 @@ export type DispatchProps = {
   listObservations: () => void,
   createObservation: (observation: Observation) => void,
   updateObservation: (observation: Observation) => void,
-  goToCategories: () => void,
-  goToObservationDetail: () => void,
   selectObservation: (observation: Observation) => void,
   updateObservationSource: () => void,
   listStyles: () => void
 };
 
 type Props = {
-  isFocused: boolean
+  isFocused: boolean,
```

Comment:
Do you mind switching this out too? New react-navigation update actually includes `isFocused` as a function on the `navigation` prop. So checking for isFocused would be `navigation.isFocused()` which returns a bool. More info on it here: https://reactnavigation.org/docs/en/navigation-prop.html

Reply:
👍 alrighty